src/components/pager/index.js

```diff
@@ -265,10 +265,9 @@ export class VlPager extends vlElement(HTMLElement) {
   }
 
   _paginationDisabledChangedCallback(oldValue, newValue) {
```

Comment:
Dit is een bug dat je hebt opgelost, en mag in de changelog! Elke change moet daar terug te vinden zijn.

Reply:
Toegevoegd aan de changelog.